e2e/app/aou-elements/xpath-defaults.ts

```diff
@@ -1,12 +1,16 @@
 import TextOptions from './text-options';
 
 function textXpathHelper(opts: TextOptions) {
+  let txt;
```

Comment:
Nit: I'd just call it `text`. Or if we can have a more descriptive name describing where the text comes from? Because I am not super following what text this variable is meant to contain.

Reply:
how about `findText`?